src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java

```diff
@@ -293,6 +293,48 @@ public void help() throws SpeechletException {
     }
 
     @Test
+    public void repeat(@Mocked ObaDao obaDao) throws SpeechletException, URISyntaxException, IOException {
```

Comment:
Thanks!
On Apr 14, 2016 12:18 AM, "Philip M. White" <notifications@github.com>
wrote:

> In src/test/java/org/onebusaway/alexa/AuthedSpeechletTest.java
> <https://github.com/OneBusAway/onebusaway-alexa/pull/56#discussion_r59661545>
> :
>
> > @@ -293,6 +293,48 @@ public void help() throws SpeechletException {
> >      }
> >
> >      @Test
> > +    public void repeat(@Mocked ObaDao obaDao) throws SpeechletException, URISyntaxException, IOException {
>
> Done.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OneBusAway/onebusaway-alexa/pull/56/files/ee9604d0d7c6ae10a8d4b0b0f14d47dd382f4553#r59661545>
>